### PR TITLE
feat(AI/Alerts): Add a transition guide for the new AI/Alerts nav (DOC-7601)

### PR DESCRIPTION
**Pull request by paperclypse**
Pushing this up as a draft.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 1]
```diff
@@ -1,15 +1,79 @@
...
   - Alerts and Applied Intelligence
   - New Relic Alerts
   - EOL
   - Transition guide
 metaDescription: 'Resources to make a successful transition as new functionalities in New Relic One replace alerting functionalities reaching end of life in 2022.'
+redirects:
+  - /docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022 
 ---
 
 Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:14]: I wonder if the 2nd paragraph and 2 bullets after "What's changing" should move up to here (after the intro paragraph) so it's easier to see? That particular Explorers Hub post is currently the main one about alerts/AI changes, so I think it could easily apply to both H2 sections.

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 2]
```diff
@@ -1,15 +1,79 @@
...
 
+## What's changed [#changed]
+
+Starting in late 2021, we've reorganized the navigation UI for alerts and applied intelligence. Although the alerts and applied intelligence navigation has changed pretty dramatically, all the features you use are still there.
+
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:18]: I'd love some kind of "why it matters" statement that expands on why we changed the UI. For me, it's very exciting that the UI is now organized in a workflow that clearly states what you can use to solve particular kinds of problems (analyze, detect. etc.).

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 3]
```diff
@@ -1,15 +1,79 @@
...
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="nav-changes"
+    title="An overview of alerts and applied intelligence nav changes"
+  >
+
+  ## Analyze [#analyze]
+
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:33]: Maybe an intro sentence about the kinds of things you might want to analyze, and, ta da! The UI helps you do exactly those things! Like, why do I care about open, closed, muted violations? What is the timeframe for most recent issues -- can I change that?

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 4]
```diff
@@ -1,15 +1,79 @@
...
+The [overview page](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page) shows your most recent open, closed, and muted violations.
+
+The **Issues & activity** page shows your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+
+## Detect [#detect]
+
+The [alert conditions (policies) page](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) lists the policies where your conditions are created and organized. Create new policies here.
+
+The [anomaly detection page](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/) lists your expanded anomaly detection configurations. You can add or edit the configurations here.
+
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:43]: I'm a new user and I have no idea what is anomaly detection or why that matters. Can you explain what you mean in a few words so I know whether I want to click the link?

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 5]
```diff
@@ -1,15 +1,79 @@
...
+  >
+
+  ## Analyze [#analyze]
+
+The [overview page](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page) shows your most recent open, closed, and muted violations.
+
+The **Issues & activity** page shows your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+
+## Detect [#detect]
+
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:39]: Same as previous. I'd love an intro sentence to excite me about this layout. If nothing else, just to clarify that the analyze section gets you rolling but you have to set up things so the system can detect them.

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 6]
```diff
@@ -1,15 +1,79 @@
...
+The [alert conditions (policies) page](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) lists the policies where your conditions are created and organized. Create new policies here.
+
+The [anomaly detection page](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/) lists your expanded anomaly detection configurations. You can add or edit the configurations here.
+
+## Correlate [#correlate]
+
+The [sources page](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources) shows your configured incident intelligence sources. You can add new sources or configure existing ones.
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:46]: maybe reword as "shows the sources you select to configure"... what? "incident intelligence" sounds like what New Relic would call it, not what I as a user would call it. If nothing else, what kinds of sources can I select? Not the whole laundry list but maybe a few standard things and/or some things our competitors cannot configure alerting on?

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 7]
```diff
@@ -1,15 +1,79 @@
...
+The **Issues & activity** page shows your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+
+## Detect [#detect]
+
+The [alert conditions (policies) page](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) lists the policies where your conditions are created and organized. Create new policies here.
+
+The [anomaly detection page](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/) lists your expanded anomaly detection configurations. You can add or edit the configurations here.
+
+## Correlate [#correlate]
+
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:45]: Same comment as before, maybe an intro sentence to clarify what exactly you're going to be able to correlate.

Comment (barbnewrelic) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:45]: And my apologies if this is strarting to smell a bit markety. :-) Feel free to ignore everything I say!

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 8]
```diff
@@ -1,15 +1,79 @@
...
+The [sources page](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources) shows your configured incident intelligence sources. You can add new sources or configure existing ones.
+
+The [decisions page](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/) shows your correlated incidents. You can review, edit, and add decisions.
+
+## Enrich & notify [#enrich]
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:50]: not sure you want to use an ampersand even though if it's in the UI...?
But I'm not quite understanding what "enrich" means here. For me, the newbie, adding muting rules would do exactly the opposite. Maybe point out the reason why muting rules are a happy thing to enrich (zero in on?) the things you care about so you can focus on something else? Like, you could set up muting rules when testing something but you want them to be on for the go-live moment, or for specific holiday shopping timeframes, or...?

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 9]
```diff
@@ -1,15 +1,79 @@
...
+
+Use [pathways](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#3-configure-pathways) to configure where your correlated issues get sent.
+
+Use [incident workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/) to automate common problems and actions related to your alerts policies.
+
+Use [incident destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/incident-intelligence-destination-examples/) to add and configure third-party services for your incidents.
+
+## Settings [#settings]
+
+Use the settings page to review and manage your current incident intelligence usage.
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:66]: This one doesn't have a link to another page. Intentional?

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 10]
```diff
@@ -1,15 +1,79 @@
...
+
+## Alerts (classic) [#alerts]
+
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:69]: I would like the intro here to say basically what you told me when we were chatting, that this is the existing UI and there's nothing wrong with that, it's just a different view while we transition the entire UI. Or something along those lines.

**Review by barbnewrelic — DISMISSED** [review]
This is great, and I esp like what you did with your UI screenshot! My review contains way too many questions and comments, and I know this is a WIP, so please don't feel like you need to do anything about all my chatter. :-)

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:14]: Excellent idea!

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:18]: Great suggestion!

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:33]: Yup, I've gone through and added some intro sentences to each of these sections.

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 4 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:43]: I added a sentence to try to clarify that. I sure hope this isn't where a new user is ending up :D

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 5 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:39]: I added one here too.

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 6 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:46]: Totally agree. I've tried to expand on this without making it a laundry list :)

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 7 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:45]: I don't think it's market-y at all. It's useful context, I think/hope!

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 8 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:50]: Muting rules are more on the notify side of things, than the enrich side of things. Personally, I feel like muting rules should go under settings :D 

Yeah, I don't like the ampersand there either, but I'm erring on the side of matching the UI, I guess.

I'm hoping that my intro sentence will help clarify this a bit.

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 9 above)
Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:66]: There isn't a docs page on this one.

**Review by barbnewrelic — APPROVED** [review]
This is so helpful, thanks for providing all this great new info!
